src/main/java/uk/gov/companieshouse/overseasentitiesapi/validation/OverseasEntitySubmissionDtoValidator.java

```diff
@@ -55,6 +58,7 @@ public Errors validateFull(OverseasEntitySubmissionDto overseasEntitySubmissionD
     private void validateFullUpdateDetails(OverseasEntitySubmissionDto overseasEntitySubmissionDto, Errors errors, String loggingContext) {
         // Method to be added to as Update journey developed
         validateFullCommonDetails(overseasEntitySubmissionDto, errors, loggingContext);
+        ownersAndOfficersDataBlockValidator.validateOwnersAndOfficers(overseasEntitySubmissionDto, errors, loggingContext);
```

Comment:
Add to Common validation and remove from registration specific part if this is now good to be added. The BO/MO work is still in progress so this may fail while we work on it.